README.md

```diff
@@ -141,6 +141,16 @@ public class Program
 }
 ```
 
+### AvroGen tool
+
+The Avro serializer and deserializer provided by `Confluent.Kafka.Avro` can be used with the `GenericRecord` class
```

Comment:
Maybe make Confluent.Kafka.Avro a link to the NuGet package?

Reply:
people find this via their ide, and it's linked above, so i think no need.